x/mint/keeper/genesis.go

```diff
@@ -12,12 +12,10 @@ func (k Keeper) InitGenesis(ctx sdk.Context, ak types.AccountKeeper, bk types.Ba
 	k.SetMinter(ctx, data.Minter)
 	k.SetParams(ctx, data.Params)
 
-	if !ak.HasAccount(ctx, ak.GetModuleAddress(types.ModuleName)) {
-		ak.GetModuleAccount(ctx, types.ModuleName)
-		totalDeveloperVestingCoins := sdk.NewCoin(data.Params.MintDenom, sdk.NewInt(225_000_000_000_000))
-		k.CreateDeveloperVestingModuleAccount(ctx, totalDeveloperVestingCoins)
-		bk.AddSupplyOffset(ctx, data.Params.MintDenom, sdk.NewInt(225_000_000_000_000).Neg())
-	}
```

Comment:
This old logic seems correct? The account would get exported in ExportGenesis?

Reply:
You are right. I assumed that this check would cover it: https://github.com/osmosis-labs/osmosis/blob/f793aaaf252cf15c81a793cc4e4fbcf8677a92ce/x/mint/keeper/keeper.go#L54-L56

On another look, it is completely unrelated